install/empty_data.php

```diff
@@ -305,7 +305,8 @@
    'default_dashboard_mini_ticket'           => 'mini_tickets',
    'admin_email_noreply'                     => '',
    'admin_email_noreply_name'                => '',
-   Impact::CONF_ENABLED                      => exportArrayToDB(Impact::getDefaultItemtypes())
+   Impact::CONF_ENABLED                      => exportArrayToDB(Impact::getDefaultItemtypes()),
+   'document_max_size'                       => '1',
```

Comment:
```suggestion
   'document_max_size'                       => Toolbox::return_bytes_from_ini_vars(ini_get('upload_max_filesize')),
```
same as in update script